packages/smart-contracts/src/lib/ethereumProxyArtifact.ts

```diff
@@ -0,0 +1,61 @@
+const artifactsEthereumProxy = require('../../artifacts/EthereumProxy/artifacts.json');
+const ARTIFACTS_VERSION: string = artifactsEthereumProxy.lastVersion;
+
+/**
+ * Retrieve the abi from the artifact of the used version
+ * @returns the abi of the artifact as a json object
+ */
+export function getContractAbi(): any {
```

Comment:
I feel sad we have to do this copy paste for every smart-contract. 
It would be nice to have a better solution in the long term.

Reply:
agree. I Jira it.